haas/api.py

```diff
@@ -661,6 +661,22 @@ def port_detach_nic(port):
     db.commit()
 
 
+@rest_call('GET', '/projects')
+def list_projects():
+    """List all projects and nodes that belong to them.
+
+    Returns a JSON array of objects representing a list of projects.
+
+    Example:  '[{"nodes": ["node1", "node2"], "name": "project1"},
+                {"nodes": ["node3", "node4"], "name": "project2"}]'
```

Comment:
I don't think so. We can write another API call to get headnode of the project.
Example:
`/projects/<project_name>/headnode`